pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Hello, I think here is a typo here. I think it should be:
```suggestion
    return x * y
```
Cheers, James